bitshares/bitshares-community-ui
# 137 amount value shortener

#### What's this PR do?
Adds Amount Value Shortener function along with two unit tests (i.e. for millions and for thousands)
#### Where should the reviewer start?
The function is placed in helpers/utils.js, and named `amountValueShortener`
The unit tests can be found in the appropriate folder used for unit tests
#### How should this be manually tested?
In the browser check out Markets component on the appropriate page
or
`npm run test:unit`
#### Any background context you want to provide?
The value shortener can be improved even more in just several minutes to accept even bigger values (i.e. trillions and much much more)
#### What are the relevant tickets?
#137 
#### Screenshots (if appropriate)
<img width="74" alt="screen shot 2018-11-01 at 20 07 12" src="https://user-images.githubusercontent.com/26275425/47870239-c07c9300-de11-11e8-99ac-0dbe89baa63d.png">

#### URL
Provide a link to your deploy from http://staging-ui.trusty.apasia.tech:8080/
http://staging-ui.trusty.apasia.tech:8080/137-amount-value-shortener

#### Questions:
-
